meta-lmp-base/recipes-samples/images/lmp-feature-alsa.inc

```diff
@@ -0,0 +1,11 @@
+# LmP specific settings for ALSA
+
+# Add user fio to audio group to control audio devices
+inherit extrausers
+EXTRA_USERS_PARAMS = "usermod -a -G audio fio"
```

Comment:
This was already done as part of https://github.com/foundriesio/meta-lmp/commit/9742ad8cae71fa87d00f4078c42ad8d932328e82

Reply:
Fixed.